config/manager.go

```diff
@@ -135,6 +135,8 @@ type Manager interface {
 	GetImagePullCriKindOverride() string
 	// GetDockerDaemonConfig returns the secret name to mount in /etc/docker.
 	GetDockerDaemonConfig() string
+	// GetDockerConfig returns the secret name to mount in /root/.docker/.
+	GetDockerConfig() string
```

Comment:
I presume you intentionally left `/root/.docker/.` prefix in the comments? But are there cases where non `config.json` is being utilized?

Reply:
updated this and the daemon bits to make it more clear (I hope :p)